src/applications/ivc-champva/10-7959f-1/tests/e2e/10-7959f-1-FMP.cypress.spec.js

```diff
@@ -97,10 +84,33 @@ const testConfig = createTestConfig(
           });
         });
       },
+      'review-and-submit': ({ afterHook }) => {
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            reviewAndSubmitPageFlow(data.statementOfTruthSignature, `Submit`);
+          });
+        });
+      },
     },
 
     setupPerTest: () => {
-      cy.intercept('POST', formConfig.submitUrl, { status: 200 });
+      cy.intercept('GET', '/v0/feature_toggles?*', mockFeatureToggles);
+      cy.intercept('POST', formConfig.submitUrl, req => {
+        cy.get('@testData').then(dataPreTransform => {
+          // Runs the test-data through transformForSubmit and compares it
+          // to the submitted form data to make sure that all outputs are present:
+          verifyAllDataWasSubmitted(
+            JSON.parse(
+              formConfig.transformForSubmit(formConfig, {
+                data: dataPreTransform,
+              }),
+            ),
+            req.body,
+          );
+        });
+        // Mock response
+        req.reply({ status: 200 });
+      });
```

Comment:
Extra mocking to catch feature toggle API request and verify the submitted data matches 1:1 to the test data input.